packages/ember-glimmer/lib/utils/iterable.js

```diff
@@ -165,28 +165,39 @@ class Iterable {
 
     if (isProxy(iterable)) {
       valueTag.update(CURRENT_TAG);
+      iterable = get(iterable, 'content');
     } else {
       valueTag.update(tagFor(iterable));
     }
 
     if (iterable === undefined || iterable === null) {
       return EMPTY_ITERATOR;
-    } else if (isEachIn(ref)) {
+    }
+
+    let typeofIterable = typeof iterable;
+
+    if (typeofIterable !== 'object' && typeofIterable !== 'function') {
```

Comment:
@krisselden I meant to ask about this. I saw this pattern (saving the typeof result into a variable) in some other places. Is this generally necessary, or could it end up doing more harm than good? (I wrote it because in the original version I did the check in several branches, but now it's just this one line I wonder if it's still needed.)